c7n/policy.py

```diff
@@ -257,19 +257,25 @@ def run(self, *args, **kw):
                 self.policy.options.region or 'default',
                 version)
 
-            s = time.time()
+            region = self.policy.options.region or 'default'
+            resource = self.policy.resource_type
+            start = time.time()
             resources = self.policy.resource_manager.resources()
-            rt = time.time() - s
+            run_time = time.time() - start
+            qty_resources = str(len(resources))
```

Comment:
can we just remove this log change, its got a lot of extraneous transient variables, and trunk has changed to include region, afaics the only delta is spacing formatting on the log string, the information conveyed is the same.